modules/simple_forms_api/app/form_mappings/vba_40_0247.json.erb

```diff
@@ -0,0 +1,28 @@
+{
+  "form1[0].#subform[0].TextField1[0]": "<%= data.dig('veteran_full_name', 'first') + ' ' + (data.dig('veteran_full_name', 'middle') || '') + ' ' + data.dig('veteran_full_name', 'last') %>",
```

Comment:
nit - will we always get a double space between first and last here if middle is not provided?

Reply:
@rhasselle-oddball Yep, we will. I thought about it and figured to actually address it more properly would make the logic on this line even more confusing so I figured it'd be ok. Do you think it's worth addressing? (I guess I figured an extra space here would be pretty harmless 🙈)

On the point about users uploading something malicious, in developing this I discovered that we use a package, [`clamav`](https://www.clamav.net/), which will handle that sort of thing. So my best guess is that this is all handled already for us but I'll definitely want to see how it works in staging (not that I have a virus-laden file on my computer sitting around to test with...).